examples/basic/variables.tf

```diff
@@ -16,36 +12,78 @@ variable "resource_group" {
   description = "String describing resource groups to create or reference"
   type        = string
   # TODO: Temp fix
-  default = "geretain-hpc-rg"
+  default = "Default"
```

Comment:
I suggest you default this to `null` and update the example to create one if `null` is used. For example -> https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/c8606d65571222af1f5078742551a2105ee45d5a/examples/basic/main.tf#L5-L11